LinkDotNet.ValidationExtensions.Tests/RangeAttributes/DynamicRangeBuiltInTypesTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using FluentAssertions;
+using Xunit;
+
+namespace LinkDotNet.ValidationExtensions.Tests.Generic;
+
+public partial class DynamicRangeBuiltInTypesTests
+{
+    // https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types
+    public static IEnumerable<object[]> TestData()
```

Comment:
That is personal preference but if you make `TestData` `private` you can move it to the bottom of the file. That leaves you with the most important stuff on top: The tests.

Reply:
Also preference:
As they are all compile time constant you can use:
[Theory]
[InlineData(...)]

Reply:
> Also preference: As they are all compile time constant you can use: [Theory] [InlineData(...)]

There are 28 items and with `[InlineData(...)]` it's going to be so long.
I moved it after the test method. ([8cffb1f](https://github.com/linkdotnet/ValidationExtensions/pull/7/commits/8cffb1f096a97f9301d4bdc180b6de74f8177f8b))
And it can't be `private` ([xUnit1016](https://xunit.net/xunit.analyzers/rules/xUnit1016))

Reply:
Yeah. That would be too much :) but nice test coverage. I like